src/typechecker.cpp

```diff
@@ -8,9 +8,10 @@
 namespace TypeChecker {
 
 TypeChecker::TypeChecker() {
-	m_meta_core.new_meta(); // 0 | value
-	m_meta_core.new_meta(); // 1 | type func
-	m_meta_core.new_meta(); // 2 | mono type
+	// NOTE: we don't care about the term data
```

Comment:
more like, "INVARIANT: terms have no arguments"

it is not that we don't care about the arguments, but that we care about the no arguments

Reply:
What change do you propose here? I had no good ideas.

Reply:
Just a bettsr comment